triton-docker

```diff
@@ -48,6 +56,18 @@ function install () {
     rm -Rf /tmp/triton-docker
     chmod +x /usr/local/bin/triton-docker-helper
 
+    # Install a good root CA store for docker-compose
+    # We'll use the directory where triton profile outputs docker certs.
+    if ! [[ -d ~/.triton/docker ]]; then
+        mkdir -p ~/.triton/docker
+    fi
+    curl -sf -o ~/.triton/docker/cacert.pem https://curl.haxx.se/ca/cacert.pem
+    cert_ret=$?
+    if (( cert_ret > 0 )); then
+        echo 'Failed to install CA certs for docker-compose. This is a curl error.'
```

Comment:
1.) Should these echo to stderr aka FD 2?

2.) Should this path exit?

3.) I can't remember if $? is unsigned... if it is, NVM. If it's not, maybe check for cert_ret != 0 instead?

Reply:
1. Fixed
2. This path does not normally exist, but docker-compose will fail without being given a sane certificate list. The default one used by python sucks.
3. `$?` is always set to the exit code of the last command. This will always be `0` on success or != `0` for error. `curl` only emits positive return codes, but I changed this from `>` to `!=`, just in case the user has something weird going on.